From: Boyuan Yang <byang@debian.org>
Date: Tue, 2 Jun 2020 22:11:52 -0400
Subject: catalog.xml: Compatibility with 1.79.1 or earlier

The docbook-xsl project changed the canonical URI since 1.79.2.
Besides changes made in debian/*.xmlcatalogs files, this patch
completes the support of the old URI so that existing software
does not break with changed registered URI.

Forwarded: not-needed
Last-Update: 2020-06-02

--- catalog.xml
+++ catalog.xml
@@ -5,4 +5,7 @@
   <rewriteSystem systemIdStartString="http://cdn.docbook.org/release/xsl/current/" rewritePrefix="./"/>
   <rewriteURI uriStartString="http://cdn.docbook.org/release/xsl/1.79.2/" rewritePrefix="./"/>
   <rewriteSystem systemIdStartString="http://cdn.docbook.org/release/xsl/1.79.2/" rewritePrefix="./"/>
+  <!-- Also support old URI of v1.79.1 or earlier -->
+  <rewriteURI uriStartString="http://docbook.sourceforge.net/release/xsl-ns/current/" rewritePrefix="./"/>
+  <rewriteSystem systemIdStartString="http://docbook.sourceforge.net/release/xsl-ns/current/" rewritePrefix="./"/>
 </catalog>
